external/fv3fit/fv3fit/emulation/layers/normalization.py

```diff
@@ -165,3 +169,23 @@ class MeanFeatureStdNormLayer(MaxFeatureStdNormLayer, FeatureMeanStd):
     """
 
     pass
+
+
+def get_norm_class(key):
```

Comment:
Based on your comment, the idea is instantiated layers returned from these classes.  I think I might just remove them from the public API and move the function to `models.py` since that's where I'm using it.